modules/apps/27-interchain-accounts/host/types/msgs_test.go

```diff
@@ -74,3 +75,71 @@ func TestMsgUpdateParamsGetSigners(t *testing.T) {
 		}
 	}
 }
+
+func TestMsgModuleQuerySafeValidateBasic(t *testing.T) {
```

Comment:
We missed this test coverage in the original PR.

Reply:
Thanks